modules/orchestrator/proc-helper-script-overview.adoc

```diff
@@ -16,7 +16,7 @@ Do not use `plugin-infra.sh` in production.
 +
 [source,terminal,subs="+attributes,+quotes"]
 ----
-curl -sSLO https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/release-${PRODUCT_VERSION}/config/profile/rhdh/plugin-infra/plugin-infra.sh # Specify the {product} version in the URL or use main
+curl -sSLO https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/release-${product-version}/config/profile/rhdh/plugin-infra/plugin-infra.sh # Specify the {product} version in the URL or use main
```

Comment:
```suggestion
curl -sSLO https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/release-{product-version}/config/profile/rhdh/plugin-infra/plugin-infra.sh # Specify the {product} version in the URL or use main
```

Otherwise, it renders as `release-$1.8`: 

<img width="827" height="324" alt="Image" src="https://github.com/user-attachments/assets/3d946c03-b3c6-4049-8606-3d260de5b731" />